docs/Userguide_login.rst

```diff
@@ -1,10 +1,22 @@
 Logging in to the cluster
 =========================
 
-To access the Mila Cluster clusters, you will need an account. Please contact
+To access the Mila Cluster clusters, you will need a MILA account. Please contact
 Mila systems administrators if you don't have it already. Our IT support service
 is available here: https://it-support.mila.quebec/
 
+You will also need to complete and return an IT Onboarding Training to get
+access to the cluster.  Please refer to the MILA Intranet for more
+informations:
+https://sites.google.com/mila.quebec/mila-intranet/it-infrastructure/it-onboarding-training
+
+**IMPORTANT** : Your access to the Cluster is granted based on your status at
+MILA (for students, your status is the same as your main supervisor' status),
+and on the duration of your stay, set during the creation of your account. The
+following have access to the cluster : **Current Students of Core Professors -
+Core Professors - Staff**
+
+
```

Comment:
Best to keep the same number of empty line between paragraph for consistency (so 2 instead of 3 in this case)